feat: permission / change table -> roles_id to role_id / view button …

HeimyJohn opened the pull request:
…in catalogue and product detail when you are admin

Commit:
feat: permission / change table -> roles_id to role_id / view button in catalogue and product detail when you are admin

XavierFigari commented on app/Http/Controllers/ProductAddController.php:
Please delete this file. It has been combined with ProductAdminController.

XavierFigari committed:
Delete app/Http/Controllers/ProductAddController.php

This file has been deleted. The "add" controller is combined with "update" in ProductAdminController.php

XavierFigari committed:
Update product.blade.php

Change comment for "update" button

XavierFigari commented:
Closes #97
